background/services/chain/index.ts

```diff
@@ -959,17 +943,16 @@ export default class ChainService extends BaseService<Events> {
     network: NetworkInterface,
     hash: string
   ): Promise<TransactionResponse | null | undefined> {
-    const provider = this.currentProvider
+    const { jsonRpcProvider } = this
```

Comment:
For consistency, we should access the `jsonRpcProvider` directly as `this.jsonRpcProvider` like is done everywhere else.